front/src/applications/editor/data/utils.ts

```diff
@@ -149,15 +145,15 @@ export function nestEntity(entity: EditorEntity, type: EditoastType): EditorEnti
   };
 }
 
-export function entityToCreateOperation(entity: EditorEntity): Operation {
+export function entityToCreateOperation(entity: EditorEntity): CreateOperation {
   return {
     operation_type: 'CREATE',
     obj_type: entity.objType,
     railjson: {
       ...entity.properties,
       id: uuid(),
     },
-  };
+  } as unknown as CreateOperation;
```

Comment:
It seems to work without `as unknown `

```suggestion
  } as CreateOperation;
```